packages/ember-application/lib/system/application.js

```diff
@@ -316,46 +295,19 @@ var Application = Namespace.extend(DeferredMixin, {
     Create a container for the current application's registry.
 
     @private
-    @method buildContainer
+    @method buildInstance
     @return {Ember.Container} the configured container
   */
-  buildContainer: function() {
+  buildInstance: function() {
     var container = this.__container__ = this.__registry__.container();
```

Comment:
@dgeb - `Ember.Application.create().__container__` is not removed here (AFAICT).  Are you suggesting that `ApplicationInstance.create().__container__` should be used instead of `ApplicationInstance.create().container`?